[2023-12-17T09:35:10.397+0700] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: test_1.check_daily_file_exists manual__2023-12-17T02:33:36.360825+00:00 [queued]>
[2023-12-17T09:35:10.401+0700] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: test_1.check_daily_file_exists manual__2023-12-17T02:33:36.360825+00:00 [queued]>
[2023-12-17T09:35:10.401+0700] {taskinstance.py:1361} INFO - Starting attempt 1 of 4
[2023-12-17T09:35:10.408+0700] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): check_daily_file_exists> on 2023-12-17 02:33:36.360825+00:00
[2023-12-17T09:35:10.410+0700] {standard_task_runner.py:57} INFO - Started process 348849 to run task
[2023-12-17T09:35:10.412+0700] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'test_1', 'check_daily_file_exists', 'manual__2023-12-17T02:33:36.360825+00:00', '--job-id', '32', '--raw', '--subdir', 'DAGS_FOLDER/pipeline_dags.py', '--cfg-path', '/tmp/tmp5hlkgiz5']
[2023-12-17T09:35:10.413+0700] {standard_task_runner.py:85} INFO - Job 32: Subtask check_daily_file_exists
[2023-12-17T09:35:10.430+0700] {task_command.py:416} INFO - Running <TaskInstance: test_1.check_daily_file_exists manual__2023-12-17T02:33:36.360825+00:00 [running]> on host phuc-ASUS
[2023-12-17T09:35:10.458+0700] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='test_1' AIRFLOW_CTX_TASK_ID='check_daily_file_exists' AIRFLOW_CTX_EXECUTION_DATE='2023-12-17T02:33:36.360825+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-12-17T02:33:36.360825+00:00'
[2023-12-17T09:35:10.464+0700] {logging_mixin.py:154} INFO - /home/phuc/Practice/DataScience/DSProject/data/raw/careerlink/careerlink_2023-12-17T02-33-38+00-00.json
[2023-12-17T09:35:10.465+0700] {python.py:194} INFO - Done. Returned value was: None
[2023-12-17T09:35:10.467+0700] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=test_1, task_id=check_daily_file_exists, execution_date=20231217T023336, start_date=20231217T023510, end_date=20231217T023510
[2023-12-17T09:35:10.504+0700] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-12-17T09:35:10.514+0700] {taskinstance.py:2778} INFO - 1 downstream tasks scheduled from follow-on schedule check
